pkg/cloud/volume_limits.go

```diff
@@ -551,3 +556,20 @@ var gpuInstanceGpus = map[string]int{
 	"p4de.24xlarge": 8,
 	"p5.48xlarge":   8,
 }
+
+// Note this table is not a reflection of how many accelerators an instance has but of how many slots their combined acclerators take up
+// VT instance type accelerators take two slots each with the exception of the vt1.24xlarge which takes 0 slots for its acclerators
+// inf1 instance types are purposely not added to this table as they are in the maxVolumeLimits table
+var acceleratorSlotsTaken = map[string]int{
```

Comment:
Can we add *where* this information is sourced from in the comment above? it will be helpful to future maintainers.

Reply:
+1, especially since AWS Docs are usually really great about re-directing dead pages to their new homes.